src/components/WorkChart.astro

```diff
@@ -0,0 +1,17 @@
+---
+type Props = {
+  img: string;
+  chartModifier: string | null;
```

Comment:
If you want a Prop to be Optional you can just declare it like this: title?: string --- The ? symbol tells TypeScript to ignore it if it's not present. Please fix.

Reply:
One more thing, the name of the Prop could be better, what exactly this do? Modify what? Since it might be just a modifier class you could change the name to className or something like that.